lib/project-view.coffee

```diff
@@ -81,7 +81,7 @@ class ProjectView extends FuzzyFinderView
         # throw ENOENT when it tries to get the realpath of all the project paths.
         # This catch block allows the file finder to still operate on the last
         # set of paths and still let the user know that something is wrong.
-        if error.code is 'ENOENT'
+        if error.code is 'ENOENT' or error.code is 'EPERM'
```

Comment:
Can definitely add a comment as to why it's there.  The test coverage is the fact it fails on Windows right now. I guess it would be possible to write a Mac test that also exercised the EPERM path but would seem to have less value once the Windows CI server runs specs?